scripts/setup.sh

```diff
@@ -43,6 +43,11 @@ function maybeInstallDependencies {
       shift
       ;;
 
+      --suite=*)
```

Comment:
Just to check, why do you need this here (if it's not needed for SHARDING, SHARD_INSTANCES, etc.)?